command-expansion-server/sql/commanding/tables/command_dictionary.sql

```diff
@@ -6,7 +6,8 @@ create table command_dictionary (
   version text not null,
 
   constraint command_dictionary_primary_key
-      primary key (id)
+      primary key (id),
+  constraint pair unique (mission,version)
```

Comment:
lets rename this to `command_dictionary_synthetic_key` like our other tables

Reply:
Hmm, I'm going to be a bit pedantic here and unpack what I think we mean by `synthetic` and `natural`.

- **natural key**: a key formed by some combination of columns that make up the data in this table
- **synthetic key**: an additional column whose sole purpose is to give each row a unique identifier

So the `pair` key is really a *natural* key, because it is formed by columns that are already in the data: (mission, version)

The key above - the `command_disctionary_primary_key` - is in fact a synthetic key, since it is based on a generated integer whose sole purpose is to give each row a unique identifier.

Take a look at the mission model sql table for an example:
https://github.com/NASA-AMMOS/aerie/blob/bcb2bae8de0357ccc8da2c2aaf76da8e7f49bf0b/merlin-server/sql/merlin/tables/mission_model.sql#L12-L15

So for the sake of consistency with the mission model table, I vote we rename `command_dictionary_primary_key` to `command_dictionary_synthetic_key`, and rename what used to be called `pair` to `command_dictionary_natural_key`.

Let me know if that doesn't make sense or if you see reasons to use different words here.